packs/data/pfs-season-1-bestiary.db/murderous-bathhouse-3-4.json

```diff
@@ -0,0 +1,119 @@
+{
+    "_id": "ORzg1jprbdWl64gA",
+    "data": {
+        "attributes": {
+            "ac": {
+                "value": 0
+            },
+            "hardness": 0,
+            "hp": {
+                "details": "",
+                "max": 0,
+                "temp": 0,
+                "value": 0
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 12
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>Linneus's grudge manifests as a haunt that isolates creatures in the bathroom and attempts to psychically drown them in one of three containers of water: the sink, the bath, or the wash bucket.</p>",
+            "disable": "<p>@Check[type:religion|dc:22](expert) to exorcise the spirit from one of the containers of water, or @Check[type:athletics|dc:23] or @Check[type:thievery|dc:23](trained) to overpower the ghostly force sealing a container and drain it. All three containers must be disabled to disable the entire haunt.</p>",
+            "isComplex": true,
+            "level": {
+                "value": 4
+            },
+            "reset": "<p>The haunt deactivates and resets after 1 minute.</p>",
+            "routine": "<p>(one action) One creature within the room must attempt a @Check[type:will|dc:23]&nbsp;save as the haunt attempts to drown them.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected and temporarily immune for 1 minute.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> A ghostly version of the creature's face appears in the water of the sink, the bath, or the wash bucket. The creature immediately experiences the sensation of being underwater, and if it can't breathe water, it must begin to hold its breath to avoid suffocation. The haunt can affect only one creature per container remaining in the haunt. If the creature leaves the room, it's immediately freed.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature immediately loses [[/r 1d4 #Actions]]{1d4} actions' worth of air.</p>"
```

Comment:
```suggestion
            "routine": "<p>(one action) One creature within the room must attempt a @Check[type:will|dc:23] save as the haunt attempts to drown them.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected and temporarily immune for 1 minute.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> A ghostly version of the creature's face appears in the water of the sink, the bath, or the wash bucket. The creature immediately experiences the sensation of being underwater, and if it can't breathe water, it must begin to hold its breath to avoid suffocation. The haunt can affect only one creature per container remaining in the haunt. If the creature leaves the room, it's immediately freed.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature immediately loses [[/r 1d4 #Actions]]{1d4} actions' worth of air.</p>"
```